.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: python
+
+python:
+  - "2.6"
```

Comment:
I would remove this. This is obsolete long ago. Sync with current support versions of Python.